tools/src/main/scala/com/nvidia/spark/rapids/tool/profiling/ProfileArgs.scala

```diff
@@ -33,8 +33,8 @@ rapids-4-spark-tools_2.12-<version>.jar /path/to/eventlog1 /path/to/eventlog2
  rapids-4-spark-tools_2.12-<version>.jar /path/to/DirOfManyEventLogs
 
 # If any event log is from S3:
-export AWS_ACCESS_KEY_ID=xxx
-export AWS_SECRET_ACCESS_KEY=xxx
+# Need to download hadoop-aws-<version>.jar and aws-java-sdk-<version>.jar firstly,
+# and then put fs.s3a.access.key and fs.s3a.secret.key in $SPARK_HOME/conf/hdfs-site.xml.
```

Comment:
S3a is a separate filesystem from HDFS. It's typically stored in core-site.xml. But you can also just configure in spark via  `spark.hadoop.fs.s3a...` settings. Environment variables should work too if EnvironmentVariableCredentialsProvider is configured. 

I suggest  that we don't go into details in our repo, and simply defer/link Hadoop/Spark docs for this.

Reply:
@gerashegalov I just added a doc link in README and removed the instructions in 2 Args.scala files. 
How is now?thx